File: universidad.java
import java.util.Scanner;

public class universidad {

	public static void main(String args[]){
		Scanner s = new Scanner(System.in);
		
		System.out.println("¿Cúal es tu nivel profesional pregrado o posgrado?");
		String nivel_profesional = s.nextLine();
		System.out.println("¿Cuál es tu promedio?");
		double promedio = s.nextDouble();
		int credito1 = 50000;
		int credito2 = 300000;
		if ("pregrado".equals(nivel_profesional) || "PREGRADO".equals(nivel_profesional)) {
			if (promedio>=4.5) {
				double cal1 = credito1*0.75*(28);
				System.out.println("El valor de tu credito es: "+cal1+" Solo cursarás 28 créditos");
			} else {
				if (promedio>=4.0 && promedio<4.5) {
					double cal2 = credito1*0.9*(25);
					System.out.println("El valor de tu credito es: "+cal2+" Solo cursarás 25 créditos");
				} else {
					if (promedio>=3.5 && promedio<4.0) {
						double cal3 = credito1*(20);
						System.out.println("El valor de tu credito es: "+cal3+" Solo cursarás 20 créditos");
					} else {
						if (promedio>=2.5 && promedio<3.5) {
							double cal4 = credito1*(15);
							System.out.println("El valor de tu credito es: "+cal4+" Solo cursarás 15 créditos");
						} else {
							if (promedio<2.5) {
								System.out.println("No podrá matricularse");
							} else {
								System.out.println("No es un valor valido");
							}
						}
					}
				}
			}
		} else {
			if ("posgrado".equals(nivel_profesional) || "POSGRADO".equals(nivel_profesional)) {
				if (promedio>=4.5) {
					double cal5 = credito2*0.8*(20);
					System.out.println("El valor de tu credito es: "+cal5+" Solo cursarás 20 créditos");
				} else {
					double cal6 = credito2*(10);
					System.out.println("El valor de tu credito es: "+cal6+" Solo cursarás 10 créditos");
				}
			} else {
				System.out.println("No es un valor valido");
			}
		}
	}


}

File: Universidad.py
nivel_profesional = str(input("¿Cúal es tu nivel profesional pregrado o posgrado?\n"))
promedio = float(input("¿Cuál es tu promedio?\n"))

credito1 = 50000
credito2 = 300000
if "pregrado"==nivel_profesional or "PREGRADO"==nivel_profesional:
	if promedio>=4.5:
		cal1 = credito1*0.75*(28)
		print ("El valor de tu credito es: ",cal1," Solo cursarás 28 créditos")
	else:
		if promedio>=4.0 and promedio<4.5:
			cal2 = credito1*0.9*(25)
			print ("El valor de tu credito es: ",cal2," Solo cursarás 25 créditos")
		else:
			if promedio>=3.5 and promedio<4.0:
				cal3 = credito1*(20)
				print ("El valor de tu credito es: ",cal3," Solo cursarás 20 créditos")
			else:
				if promedio>=2.5 and promedio<3.5:
					cal4 = credito1*(15)
					print ("El valor de tu credito es: ",cal4," Solo cursarás 15 créditos")
				else:
					if promedio<2.5:
						print ("No podrá matricularse")
					else:
						print ("No es un valor valido")
else:
	if "posgrado"==nivel_profesional or "POSGRADO"==nivel_profesional:
		if promedio>=4.5:
			cal5 = credito2*0.8*(20)
			print ("El valor de tu credito es: ",cal5," Solo cursarás 20 créditos")
		else:
			cal6 = credito2*(10)
			print ("El valor de tu credito es: ",cal6," Solo cursarás 10 créditos")
	else:
			print ("No es un valor valido")